src/components/Layout/Layout.jsx

```diff
@@ -1,9 +1,18 @@
-import React from 'react'
+import React, {Suspense} from 'react'
+import Header from "../Header/Header.jsx";
+import Footer from "../Footer/Footer.jsx";
+import Loader from "../Loader/Loader.jsx";
+import {Outlet} from "react-router-dom";
 
 const Layout = () => {
-  return (
-    <div>Layout</div>
-  )
+    return (
+        <>
+            <Header/>
+            <Suspense fallback={<Loader />}>
+                <Outlet />
+            </Suspense>
+            <Footer/>
+        </>)
```

Comment:
тут потрібно добавити toaster після footer
```suggestion
        <Toaster position='top-right' reverseOrder={false} />)
```